tests/testthat/test-module_nested_tabs.R

```diff
@@ -445,8 +447,9 @@ testthat::test_that("srv_teal_module: teal_module receives transformed data with
       datasets()$set_filter_state(teal_slices(teal_slice("mtcars", "cyl", selected = "6")))
       session$flushReact()
       out <- module_out()
-      testthat::expect_identical(out()[["iris"]], dplyr::filter(iris, Species == "virginica") |> head(6))
-      testthat::expect_identical(out()[["mtcars"]], dplyr::filter(mtcars, cyl == 6) |> head(6))
+      # subset keeps the old dataframe row.names where dplyr::filter used in the app resets row.names to 1:nrow()
+      testthat::expect_identical(out()[["iris"]], subset(iris, Species == "virginica") %>% `rownames<-`(1:50) %>% head(6))
```

Comment:
```suggestion
      testthat::expect_identical(out()[["iris"]], subset(iris, Species == "virginica") %>% magrittr::set_rownames(1:50) %>% head(6))
```

Reply:
@averissimo this is good addition, but I had a call the other day where I understood we rather remove `magrittr` from `teal` codebase

Reply:
If that's the choice then merge away! Other than create a function only for this, there are not many alternatives.